### DISCUSSION TITLE
Text Preview Tab

### DISCUSSION THREAD

**Discussion by tesseract2021** [vkbo/novelWriter#1037] [General]
I just installed novelWriter and finding my around. I really like what I see; enough tools to serve my needs without being so 'overweight' it's unusable.
Is there a way to split the markdown editor window with a plain text view?

Thanks,

**Comment by vkbo** [comment]
Absolutely. Just right click and select "view document" (or middle mouse button click) a document in the project tree, and it should open in a split panel to the right of the editor. You can use it as a preview if you wish, but you can open any document there. It is read only.

See: https://novelwriter.readthedocs.io/en/stable/usage_interface.html#editing-and-viewing-documents

I'm glad you find novelWriter useful :)

**Comment by tesseract2021** [comment]
Thanks! I finally found the manual. The more I use the program, the more I like it! It''s simple yet has more than enough tools for a involved project; it's INTUITIVE! A quality often lost these days. I am a retired electrical engineer and 'see' what you were thinking developing this project. I'm writing a novel and have a secondary project writing an in-depth start-up procedure.

One concern  I did want  to mention is the Qt 3 will not run on any of the Ubuntu 'flavors' >18.XX. Only lib4 is being installed. I haven't tried it on my Debian 11 box. I'll look at the Qt install on it.

Thank you for this nice, powerful piece of software! Now it sure would be nice to solve that quantum physics and general relativity.......

_The only difference between playing around and science is keep records!_

**Reply by vkbo** [reply]
> One concern I did want to mention is the Qt 3 will not run on any of the Ubuntu 'flavors' >18.XX. Only lib4 is being installed. I haven't tried it on my Debian 11 box. I'll look at the Qt install on it.

I'm not following. Qt 5.3 or greater is required, and runs on all recent Ubuntu releases. From next minor release (1.7), 18.04 will no longer be supported due to it having a deprecated Python version (3.6).

**Reply by tesseract2021** [reply]
I misspoke in the previous post. Qt 4 is supported but not the 32bit libraries.
A link to the question I asked the Ubuntu support community

https://ubuntuforums.org/showthread.php?t=2473661

Hope this might lend some insight.

**Reply by vkbo** [reply]
Not really, because I don't get how this connects to novelWriter and the Debian/Ubuntu packages I've made.

Edit: I develop novelWriter on Debian 11, and always test-install the deb packages on the same computer and usually also on Ubuntu 20.04 and latest Ubuntu. (... and Windows 10 by the way.)
